src/main/kotlin/com/terraformation/backend/funder/db/FundingEntityStore.kt

```diff
@@ -64,10 +65,12 @@ class FundingEntityStore(
                   groupRecords
                       .filter { it[PROJECTS.ID] != null }
                       .map { record ->
-                        ExistingProjectModel(
-                            id = record[PROJECTS.ID]!!,
-                            name = record[PROJECTS.NAME]!!,
-                            organizationId = record[PROJECTS.ORGANIZATION_ID]!!,
+                        val projectId = record[PROJECTS.ID]!!
+                        val variableValues =
+                            acceleratorProjectVariableValuesService.fetchValues(projectId)
+                        ProjectAcceleratorDetailsModel(
+                            projectId = projectId,
+                            dealName = variableValues.dealName,
                         )
                       })
```

Comment:
I don't love fetching values inside the map function, so I'm open to suggestions here. 
In case context changes anything here, most funding entities will only have one project (though each project is possible and likely to be associated with multiple entities), and entities that have more than one project will still likely have a small number.

Reply:
I think the other problem with this is that it's fetching a bunch of variable values unnecessarily. It's definitely possible to fold "get the latest value of the 'deal name' variable for this project" into the query, perhaps in the form of a subquery. It'd break encapsulation a bit but we're already joining with non-funding-entity-related tables so maybe that's not too bad.

That said, we're still planning to move/copy the deal name to the project accelerator details table, right? If so, then this change will be short-lived (I guess it'd never even be executed by actual end users) and presumably we'd instead join with that table in the query here to get the value.